docs/deployment/cumulus_api_launchpad_authentication.md

```diff
@@ -0,0 +1,26 @@
+---
+id: cumulus_api_launchpad_authentication
```

Comment:
need to add this page to sidebars website/sidebars.json

Reply:
need to reference this page from upper level document

Reply:
need to update https://github.com/nasa/cumulus/edit/master/docs/deployment/README.md
line 48, line 198 example